pkg/selinux/selinux-client.js

```diff
@@ -128,12 +140,28 @@ export function init(statusChangedCallback) {
                 .then(output => {
                     output = output.split("~~~~~");
                     status.shell = output[0];
+                    let formatted_shell = "";
+                    output[0].split("\n").forEach(e => {
+                        const items = e.split(" ");
+                        if (items.length === 2 && items[1] == "-D")
+                            formatted_shell += "    semanage " + e + "\n";
+                    });
                     status.modifications = [];
+                    status.ansible = `
+- name: Clean all existing selinux rules
```

Comment:
I honestly don't have a good gut feeling about that, one could argue either way. I suggest contacting the Ansible and SELinux developers and gauge their opinion? If there is no clear winner, let's leave it in -- it's much easier for an admin to drop that task than to add it.